bin/main.ts

```diff
@@ -1,17 +1,49 @@
-import { App } from '@aws-cdk/core';
-import * as ssp from '@aws-quickstart/ssp-amazon-eks';
-import { MyFluentBitAddOn } from '../dist';
+import { App } from 'aws-cdk-lib';
+import { EksBlueprint } from '@aws-quickstart/eks-blueprints';
+import { SnykMonitorAddOn } from '../dist';
+import { pipeline } from '../ci/pipeline';
 
 const app = new App();
 
-ssp.EksBlueprint.builder()
-    .addOns(new ssp.MetricsServerAddOn)
-    .addOns(new ssp.ClusterAutoScalerAddOn)
-    .addOns(new ssp.addons.SSMAgentAddOn) // needed for AWS internal accounts only
-    .addOns(new ssp.SecretsStoreAddOn) // requires to support CSI Secrets
-     .addOns(new MyFluentBitAddOn({
-         cloudWatchRegion: 'us-east-2',
-         //licenseKeySecret: 'my-addon-license', // if you set it, make sure there is a secret named my-addon-license-key in the target region
-         namespace: 'my-addon-namespace'
-     }))
-     .build(app, 'my-extension-test-blueprint');
+// load required parameters from the environment and validate them
+const account = process.env.CDK_DEFAULT_ACCOUNT!; // e.g. 492635582501
+const region = process.env.CDK_DEFAULT_REGION! || 'us-east-1';
+const stackID = process.env.STACK_ID || 'snyk-monitor-eks-blueprints-addon';
+const integrationId = process.env.INTEGRATION_ID; // e.g. abcd1234-abcd-1234-abcd-1234abcd1234
+if (!inputsAreValid()) {
+    console.log("Inputs are invalid. Exiting...");
+    process.exit(1);
+}
+const stackProps = { env: { account, region } };
+
+// deploy EKS with the Snyk Monitor addon
+EksBlueprint.builder()
+    .addOns(new SnykMonitorAddOn({
+        integrationId: integrationId,
+        version: "1.87.2",
+        values: {}
+    }))
+    .build(app, stackID, stackProps);
+
+// check each input value for correctness
+function inputsAreValid(): boolean {
```

Comment:
This could be improved later by adding checks around the string pattern these variables use, maybe we can add to the some day backlog. 
Pseudo-code:

```ts
if (!process.env.integrationID) {
  console.log("...")
} else {
  const integrationIDPattern = new RegExp("(?:\s|^)([^*\-]{8})(-.{4}){3}-([a-zA-Z0-9_]){12}");
  let integrationIDTest : boolean = integrationIDPattern.test(process.env.integrationID);

  integrationIDTest ?: console.log("Hey, that's not an Integration ID!")
}
```

Also, we're calling `process.env...` here even though we've assigned variables for this purpose up above in the global scope.

Reply:
Definitely, could be improved. I cleaned it up a bit in [b3dabdf](https://github.com/snyk-partners/snyk-monitor-eks-blueprints-addon/pull/1/commits/b3dabdf36b138b94f19427dfa377828658a26ac9). I used `process.env` a second time in certain places because of the possibility that an environment variable is set to an empty string.